code/game/machinery/doors/brigdoors.dm

```diff
@@ -31,6 +31,28 @@
 	maptext_height = 26
 	maptext_width = 32
 
+/obj/machinery/door_timer/proc/print_report()
+ 	var/logname = input(usr, "Name of the guilty?","[id] log name")
+ 	var/logcharges = input(usr, "What have they been charged with?","[id] log charges")
+
+ 	if(!logname || !logcharges)
+  		return 0
+
+ 	for(var/obj/machinery/computer/prisoner/C in prisoncomputer_list)
+ 			var/obj/item/weapon/paper/P = new /obj/item/weapon/paper(C.loc)
+ 			P.name = "[id] log - [logname]"
+ 			P.info =  "<center><b>[id] - Brig record</b></center><br><hr><br>"
+ 			P.info += "<center>[station_name()] - Security Department</center><br>"
+ 			P.info += "<center><small><b>Admission data:</b></small></center><br>"
+ 			P.info += "<small><b>Log generated at:   </b>		[worldtime2text(world.time)]<br>"
+ 			P.info += "<b>Detainee:</b>		[logname]<br>"
+ 			P.info += "<b>Duration:</b>		[timetoset/60/10] minute(s)<br>"
+ 			P.info += "<b>Charge(s):</b>	[logcharges]<br>"
+ 			P.info += "<b>Arresting Officer:</b>		[usr.name]<br><hr><br>"
+ 			P.info += "<small>Disclaimer: This is an automatic generated log file upon activation of a cell timer. Please store this file and save it for record purposes.</small>"

```

Comment:
Maybe change this to "This is an automatically generated log file"? Not sure about the truly correct grammar, but it feels more natural.

Alternatively, you could make it "This log file was generated automatically upon activation of a cell timer." which also sounds more natural.